scripts/mixins/animated_weapon.lua

```diff
@@ -0,0 +1,22 @@
+-- Animated Weapon (Dynamis) mixin
+
+require("scripts/globals/mixins")
+
+g_mixins = g_mixins or {}
+g_mixins.families = g_mixins.families or {}
+
+g_mixins.families.animated_weapon = function(weaponMob, options)
+    weaponMob:addListener("ENGAGE", "ENGAGE_ANIMATED", function(mob, target)
+        if mob:getAnimationSub() == 3 then
```

Comment:
huh, is that how that works? interesting

Reply:
Honestly not sure how this works I just replicated the existing logic